.pre-commit-config.yaml

```diff
@@ -48,14 +48,17 @@ repos:
       - id: mdformat
         name: MD formatting
         additional_dependencies:
-          - mdformat-gfm
-          - mdformat-frontmatter
-          - mdformat-mkdocs
+          - mdformat-admon>=1.1.3
+          - mdformat-frontmatter>=2.0.8
+          - mdformat_tables>=0.4.1
+          # # PLANNED: defer additional plugins (black/beautysh/etc.) to minimize diff
+          # #   Note: 'mdformat-mkdocs[recommended]' includes admon, frontmatter, and tables
+          # - mdformat-mkdocs[recommended]>=1.2.0
```

Comment:
Note, there is a dependency conflict between the latest `mdformat-frontmatter` which has fixes for wrapping and the `mdformat-gfm` plugin, which is pending a release: https://github.com/hukkin/mdformat-gfm/issues/31https://github.com/hukkin/mdformat-gfm/issues/31

Once `mdformat-gfm` is released, you can switch to `mdformat-mkdocs` for the latest versions of everything

Reply:
Oh this looks really nice! Excited for this release.

Reply:
Still blocked by the pending release of `mdformat-gfm`, but I can use a pinned version from git and have made substantial improvements to support content-tabs